enumerables.rb

```diff
@@ -0,0 +1,97 @@
+module Enumerable
+  def my_each()
+    i = -1
+    length.times do
+      i += 1
+      yield self[i]
+    end
+  end
+
+  def my_each_with_index()
+    i = -1
```

Comment:
* [ ] `my_each_with_index`  should be identical and returns the same thing as ruby's [`my_each_with_index`](https://ruby-doc.org/core-2.6.4/Array.html#method-i-each). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/my_each_with_index.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-2))
1.  ```p [1,2,3,4,5].my_each_with_index{|a| a}```  
should return same results as
 ```p[1,2,3,4,5].my_each_with_index{|a| a} ```
2. ```p [1,2,3,4,5].my_each_with_index ```  should return same results as  ``` p [1,2,3,4,5].each_with_index ```